src/amount.h

```diff
@@ -16,6 +16,8 @@ typedef int64_t CAmount;
 static const CAmount COIN = 100000000;
 static const CAmount CENT = 1000000;
 
+extern const std::string CURRENCY_UNIT;
+
 /** No amount larger than this (in satoshi) is valid.
```

Comment:
s/satoshi/zatoshi?

Reply:
There are other PRs that handle these changes. This PR solely focuses on the currency unit.

Reply:
ok